my_project/make_some_prints.py

```diff
@@ -1,3 +1,3 @@
 print('start')
-print('hello world')
+print('hello universe')
```

Comment:
Moment Mal! Kannst du das noch mal erklären?

Reply:
Ich hätte lieber das

```suggestion
print('hello all universes')
```

Reply:
Ah ja gute Idee wir nehmen einfach gleich alle Universen.